### ISSUE TITLE
Loss NAN

### ISSUE THREAD

**Issue by itbear-zane** [VITA-Group/NeuralLift-360#12]
HI, there. Thank you for your contribution. After training for a few epochs (using the training example `cabin`, no more changes), loss = NaN ocurred. `loss=nan (nan), lr=0.000100: :  38% 38/100 [01:47<03:04,  2.97s/it]WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.
WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.
WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.
WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.
WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.
WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.
WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.
WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.
WARNING:tensorboardX.x2num:NaN or Inf found in input tensor.`.
